docs/articles/nunit-engine/extensions/AvailableExtensions.md

```diff
@@ -0,0 +1,34 @@
+---
+uid: availableengineextensions
+---
+
+# Available NUnit Engine Extensions
+
+The NUnit team offers several extensions for the engine. We'll also attempt to highlight other extensions from the community.
```

Comment:
Maybe we should leave other extensions from the community for another issue. We don't want to promise it unless somebody is available to curate the collection. The engine project hasn't had any commits for four months, so it's pretty clear we don't have much time available to put into this. Would you take it on yourself?

Also, if we decide to do it, we have to establish exactly what we are doing... Just publishing info from extension authors? Testing them and certifying they work? etc.

I did this once before, about 15 years ago, for NUnit Addins, an old extensibility API for the V2 framework. I published them on the wiki and just said "nothing guaranteed" although I tested them with the current version of NUnit at the time I got them. After a while, I abandoned the effort. Engine extensions are more complicated, since they may work with one runner and not another... even with one package type (nuget, chocolatey, etc.) and not another.

Reply:
I'm fine with whatever language moves things forward and gets things merged. These are living and we can change direction at any time. If we know of extensions, we can list them. If others come to light, anyone can make a PR. I see it as best effort. If there are no extensions we know of now, I'm fine if we just remove that language.

Reply:
I'm not against this... just think it requires some pre-planning. I even have an extension I'd offer myself if we do it.

Just thought I'd add that. :-)

Reply:
I don't know of any. We could search nuget.org, of course. I agree with your "best effort" but I've been burned before, where people took the listing as some kind of guarantee and complained when it stopped working. So, IMO, it's good to spell out what level of guarantee (or non-guarantee) we offer and that's not a decision I'd make on my own. My thought would be to find someone who wants to do it and let them spell out what they are willing to support. If nobody wants to put any time in, then it's clearly not something we can do.

FWIW I'd be happy if you proposed doing it yourself with some kind of "best-effort, nothing guaranteed" statement. But I know you're kind of busy too.

Reply:
I'm not worried about guarantees or the appearance of such. Let's move forward with whatever ones we're comfortable adding, or scrap the language and leave it for another conversation. 👍

Reply:
No need to debate it here I think as there's no "right" answer. You're the sole judge of what you will publish in the docs, but I'm advising that I at least think the project as a whole needs to be concerned about specifying what our responsibility is if we make such a listing and about our ability to deliver if we promise to make one.

BTW, since the entire PR is WIP, I'll just make a comment on the issue itself when my own additions are ready to review.